pkgs/development/python-modules/qcodes/default.nix

```diff
@@ -79,14 +77,12 @@ buildPythonPackage rec {
     jsonschema
     matplotlib
     numpy
-    opencensus
-    opencensus-ext-azure
```

Comment:
Could you add these dependencies to `optional-dependencies`?
https://github.com/microsoft/Qcodes/compare/refs/tags/v0.44.1...v0.45.0#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711

Reply:
Done (along other optional dependencies).